libresolv.so.2 is dynamically loaded by most
binaries which attempt domain name resolution.
Without this /etc/resolv.conf is ignored.

Not sure how normally the sdk works without it.

This fixes things like `ping wp.pl` not working
with messages `bad address`. I found this with
`strace`.
--- a/hostTools/libcreduction/Makefile
+++ b/hostTools/libcreduction/Makefile
@@ -91,6 +91,7 @@ $(warning "32-bit: $(ARCH) $(INSTALL_SUF
 ifeq ($(LIBC),glibc)
   # Libs which are loaded via dlopen()
   EXTRALIBS32 := libnss_dns.so.2 libnss_files.so.2
+  EXTRALIBS32 += libresolv.so.2
   # Dynamic linker
   ifeq ($(strip $(ARCH)),arm)
       LINKER32 := ld-linux.so.3
